seqr/views/utils/dataset_utils.py

```diff
@@ -385,9 +374,18 @@ def _load_rna_seq(model_cls, file_path, user, mapping_file, ignore_extra_samples
     if missing_cols:
         raise ValueError(f'Invalid file: missing column(s) {", ".join(sorted(missing_cols))}')
 
+    sample_id_to_tissue_type = {}
     for line in tqdm(f, unit=' rows'):
         row = dict(zip(header, _parse_tsv_row(line)))
-        for sample_id, row_dict in parse_row(row, sample_id_to_tissue_type=sample_id_to_tissue_type):
+        for sample_id, row_dict in parse_row(row):
+            tissue_type = TISSUE_TYPE_MAP.get(row_dict.pop(TISSUE_COL, None), None)
+            if sample_id in sample_id_to_tissue_type:
+                prev_tissue_type = sample_id_to_tissue_type[sample_id]
+                if tissue_type != prev_tissue_type:
+                    raise ValueError(f'Mismatched tissue types for sample {sample_id}:'
```

Comment:
this needs to maintain previous behavior of skipping the sample, continuing to load the rest of the file, and showing the user a warning saying which samples were skipped

Reply:
Sounds good.